src/Milky/Checker.cs

```diff
@@ -233,7 +233,7 @@ private void OutputCombo(Combo combo, CheckResult checkResult)
 
                     if (_outputSettings.GlobalOutput)
                     {
-                        string globalOutputPath = Path.Combine(Path.Combine(Directory.GetParent(_outputSettings.OutputDirectory).FullName, "Global"), outputFile + ".txt");
+                        string globalOutputPath = Path.Combine(Path.Combine(Directory.GetParent(_outputSettings.OutputDirectory ?? string.Empty).FullName, "Global"), outputFile + ".txt");
```

Comment:
This will still crash with `string.Empty`.

`System.ArgumentException: Path cannot be the empty string or all whitespace. (Parameter 'path')`